### ISSUE TITLE
FR: way to specify labels at page saving time on android

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [omnivore-app/omnivore#2230]
When saving a page on android: I think this popup should contain a tickbox + field way to add labels directly without having to interrupt the browsing experience (i.e. going to the app to set it)

![image](https://github.com/omnivore-app/omnivore/assets/26625900/52af5944-77ed-4929-b77d-a686f6d9868f)

**Comment by hemanti0503** [comment]
I second this. Or at the very least provide a button to go to the just-saved article in the app to assign labels here.

**Comment by dairefagan** [comment]
> When saving a page on android: I think this popup should contain a tickbox + field way to add labels directly without having to interrupt the browsing experience (i.e. going to the app to set it)
> 
> ![image](https://private-user-images.githubusercontent.com/26625900/239733103-52af5944-77ed-4929-b77d-a686f6d9868f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE0NTg4ODksIm5iZiI6MTcwMTQ1ODU4OSwicGF0aCI6Ii8yNjYyNTkwMC8yMzk3MzMxMDMtNTJhZjU5NDQtNzdlZC00OTI5LWI3N2QtYTY4NmY2ZDk4NjhmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjAxVDE5MjMwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA3YWM4MWVhZDEyNTVlMzU0NjJmNGY1Y2MyZjYzMDcxMzJhYjFiYjljOGVjYTEwYWQxYzUzYWEzMzM4NWRmZmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.y7FIdbUL3ozPf-tWkudxRN7UT8oUJY6tZIqqe8WNkPc)

Agreed, we should not have to leave the browser to add labels, or if we have to it would at least be easier if pressing the notification brought us directly to the new save as @hemanti0503 has suggested.

Instead of this I just see a small toast/banner at the bottom of the screen for a few seconds without a dismiss button, though I guess it may just be a newer version?

**Comment by macdonaldster** [comment]
Yes, I came here trying to figure out if I was missing something. :) I'd love to just be able to open the article easily in Omnivore (Android app).

I can understand not prioritizing this request since it takes time for the app to produce a parsed version of the article so jumping right over would ultimately lead to complaints about the app content not being ready...

One usage pattern that might work is once the share has been done and the article is ready, an android notification could appear allowing the newly shared article to be opened. 

I suspect most people don't want to interrupt their reading to tag which is why it would be nice to do in browser but a persistent notification gives that ability to return to the shared article later (without having to explicitly remember to do that later) to tag it and gets around any parsing/archiving delays.
